src/DependencyInjection/Configuration.php

```diff
@@ -487,7 +487,7 @@ public function getConfigTreeBuilder()
 
                                 'bundles/sonatacore/vendor/select2/select2.min.js',
 
-                                'bundles/sonataadmin/vendor/admin-lte/dist/js/app.min.js',
+                                'bundles/sonataadmin/vendor/admin-lte/dist/js/adminlte.min.js',
```

Comment:
this is a BC-break ; can't we keep the same name ?

Reply:
It depends if we want a direct copy & paste of the original source or if we want to keep BC for the assets too.

Reply:
I agree it would be convenient to copy paste.
But if someone use `remove_javascript` to remove this js ; it won't work if we change the name of the file...

Maybe we should keep the same name in 3.x and rename in 4.x